docs/contributing/docs-templates.md

```diff
@@ -103,64 +116,58 @@ Provide at least one example of how the task gets accomplished. A code snippet i
 
 //See this [Markdown Cheatsheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code) on how to format code examples
 
-### Gatsby advantages
+## Gatsby advantages
 
 Does Gatsby address this topic uniquely in some way? If so, state the unique advantages Gatsby provides to the user.
 
 If there are disadvantages Gatsby has, state those here as well and any known bugs or issues the Gatsby community is working on.
 
-### Other resources
+## Other resources
 
 If there are other resources you think readers would benefit from or next steps they might want to take after reading your article, add
 them at the bottom in an "Other Resources" section. You can also mention here any resources that helped you write the article (blogposts, outside tutorials, etc.).
 
-- Link to a blogpost
+- Link to a blog post
 - Link to a YouTube tutorial
 - Link to an example site
 - Link to source code for a live site
 - Links to relevant plugins
 - Links to starters
 ```
 
-## Guide overview
+## Reference guide overview
 
 ### Near-perfect example of a guide overview
 
 [Deploying and Hosting](/docs/deploying-and-hosting/)
 
 ### What should a guide overview be about?
 
-Each overview should give a short overview of its section of the guides and ideally fit “above the fold” (the reader can see the whole guide overview article on desktop without scrolling).
-
-The goal: by skimming over the list of guide overview articles, a person new to Gatsby should get a good overview of what functionality Gatsby is capable of and what they can accomplish with Gatsby.
+Each overview should give a short introduction of its section of the guides and list the relevant subtopics.
```

Comment:
maybe include the [[guides]] things? Would that need to be renamed too?

Reply:
That's a good idea! We could also link to docs component information once we have it. related to #11946